lib/qwerty/style.rb

```diff
@@ -0,0 +1,28 @@
+module Qwerty
+  class Style < Ruote::Participant
+    def on_workitem
+      workitem.fields[:style] = {
+         :font => 'Arial',
+         :font_size => set_font_size(workitem.fields['text']['hadith_simple']['num_char']),
```

Comment:
Line is too long. [91/80]<br>Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.